src/vampy/__init__.py

```diff
@@ -8,21 +8,27 @@
 from .automatedPostprocessing import postprocessing_common
 from .automatedPostprocessing import visualize_probes
 # Imports from pre-processing
-from .automatedPreprocessing import DisplayData
-from .automatedPreprocessing import ImportData
-from .automatedPreprocessing import NetworkBoundaryConditions
-from .automatedPreprocessing import ToolRepairSTL
-from .automatedPreprocessing import automated_preprocessing
-from .automatedPreprocessing import preprocessing_common
-from .automatedPreprocessing import simulate
-from .automatedPreprocessing import visualize
-from .automatedPreprocessing import vmtk_pointselector
+try:
+    from .automatedPreprocessing import DisplayData
+    from .automatedPreprocessing import ImportData
+    from .automatedPreprocessing import NetworkBoundaryConditions
+    from .automatedPreprocessing import ToolRepairSTL
+    from .automatedPreprocessing import automated_preprocessing
+    from .automatedPreprocessing import preprocessing_common
+    from .automatedPreprocessing import simulate
+    from .automatedPreprocessing import visualize
+    from .automatedPreprocessing import vmtk_pointselector
+except ModuleNotFoundError:
+    print("WARNING: morphMan is not installed, automated preprocessing is not available")
 # Imports from simulation scripts
-from .simulation import Artery
-from .simulation import Atrium
-from .simulation import Probe
-from .simulation import Womersley
-from .simulation import simulation_common
+try:
+    from .simulation import Artery
+    from .simulation import Atrium
+    from .simulation import Probe
+    from .simulation import Womersley
+    from .simulation import simulation_common
+except ModuleNotFoundError:
+    print("Oasis is not installed, running CFD is not available")
```

Comment:
I see there one more line here that should be updated like the others. Otherwise it looks good.

Reply:
thank you! Now everything should be fixed.